src/lib/php/Application/LicenseCsvImport.php

```diff
@@ -228,7 +229,7 @@ private function updateLicense($row, $rfPk)
       $extraParams[] = "rf_spdx_id=$" . count($param);
       $log .= ", updated SPDX ID";
     }
-    if (!empty($row['text']) && $row['text'] != $oldLicense['rf_text']) {
+    if (!empty($row['text']) && $row['text'] != $oldLicense['rf_text'] && strlen($oldLicense['rf_text']) < $maxAllowedChars) {
```

Comment:
Should check `"The maximum number of characters per cell in a CSV file(32,767) exceeded, Please edit this license text in UI"` string instead here.

Reply:
was thinking that in case of edits to same text would modify the actual licenses text. As it is only checking the old license text/ previous text i have the same check ..

do you think there are any drawbacks ??

Reply:
Yes, in current setup, the import will update the license text to `"The maximum number of characters per cell in a CSV file(32,767) exceeded, Please edit this license text in UI"` wherever it sees it.
If the text is longer than the limit, and fossology can read it, then there is no issue right?